Package.swift

```diff
@@ -19,7 +19,7 @@ import PackageDescription
 let package = Package(
     name: "SPI-Server",
     platforms: [
-        .macOS(.v13)
+        .macOS(.v15)
```

Comment:
macOS 15 is required for typed throw runtime support due to our use of

```
        @Dependency(\.github.fetchMetadata) var fetchMetadata
```

below.